src/Microsoft.TemplateEngine.Cli/HelpAndUsage/TemplateDetailsDisplay.cs

```diff
@@ -54,10 +56,10 @@ private static void ShowTemplateDetailHeaders(IReadOnlyList<ITemplateInfo> templ
             HashSet<string> languages = new HashSet<string>();
             foreach (ITemplateInfo templateInfo in templateGroup)
             {
-                string templateLanguage = templateInfo.GetLanguage();
+                string? templateLanguage = templateInfo.GetLanguage();
                 if (!string.IsNullOrWhiteSpace(templateLanguage))
                 {
-                    languages.Add(templateLanguage);
+                    languages.Add(templateLanguage!);
```

Comment:
Can we switch to .NET 6 instead of .NET Standard 2.0 for CLI project so we can at least stop in this project putting this `!`?

Reply:
https://github.com/dotnet/templating/pull/3158